src/main/diagnostics/steps/step7.performance.ts

```diff
@@ -31,7 +31,6 @@ const run = async (logger: MainLogger): Promise<DiagnosticStepResponse> => {
                     path: heapSnapshotFilepath,
                     success: process.takeHeapSnapshot(heapSnapshotFilepath),
                 },
-                IOCounters: process.getIOCounters(),
```

Comment:
are we ok with not having I/O stats anymore?

Reply:
Unfortunately we don't have much choice, Chromium removed them on us.

Reply:
Could be security related hence got removed

Reply:
Curious, where are these stats stored?

Reply:
You can get them from the Help menu > Run diagnostics